model/user.go

```diff
@@ -2,17 +2,21 @@ package model
 
 import (
 	"github.com/jinzhu/gorm"
+	"github.com/nu7hatch/gouuid"
 	"godemo/database"
 	"golang.org/x/crypto/bcrypt"
 )
 
+// ユーザ情報を保持するモデル
 type User struct {
 	gorm.Model
 
 	Email    string `sql:"not null;unique_index"`
+	Token    string `sql:"not null"`
 	Password string `sql:"not null"`
 }
 
+// 生パスワードを与えるとハッシュ化したパスワードを返却する
```

Comment:
comment on exported function PasswordHash should be of the form "PasswordHash ..."